projects/hslayers/src/components/add-data/url/wmts/add-data-url-wmts.html

```diff
@@ -2,20 +2,13 @@
     <hs-add-data-common-url type="wmts" [(url)]="hsAddDataUrlWmtsService.url" (connect)="connect()">
     </hs-add-data-common-url>
 
-    <div class="row d-flex justify-content-center btn-group w-75 m-auto">
-        <button [hidden]="!hsAddDataUrlWmtsService.loadingInfo"
-            class="btn btn-primary btn-block w-75">{{'ADDDATA.CATALOGUE.loading' | translate}}<img class="ml-2"
-                [src]="hsUtilsService.getAjaxLoaderIcon()" /></button>
-        <button [hidden]="!hsAddDataUrlWmtsService.loadingInfo" class="btn btn-secondary"
-            (click)="hsAddDataService.cancelUrlRequest.next()">&#10006;
-        </button>
-    </div>
+    <hs-common-url-loading-data [loadingInfo]="hsAddDataUrlWmtsService.loadingInfo"></hs-common-url-loading-data>
     <div *ngIf="hsAddDataUrlWmtsService.showDetails && !hsAddDataUrlWmtsService.loadingInfo" class="card">
         <ul class="list-group">
             <li class="list-group-item hs-ows-layerlistheading clearfix" style="background: var(--white);">
                 <div>{{hsAddDataUrlWmtsService.title}}</div>
             </li>
-            <li *ngFor="let layer of hsAddDataUrlWmtsService.services; let index = index"
+            <li *ngFor="let layer of hsAddDataUrlWmtsService.services; trackBy:( 'Identifier' | trackByProperty ); let index = index"
```

Comment:
Where does Identifier come from? Will it always be present?

Reply:
Shoud be present in getCapabilities response as its in spec
![image](https://user-images.githubusercontent.com/44566616/133241940-a91ff057-24d3-4be2-bc9d-548a8bd7f8f1.png)